rules/game_process.tex

```diff
@@ -89,7 +94,8 @@ \subsection{Robot States}
 The colors corresponding to the game states are:
 \begin{itemize}
   \item \texttt{Unstiff}: Blue-Blinking
-  \item \texttt{Initial}: Off
+  \item \texttt{Setup}: Off
+  \item \texttt{Initial}: Cyan or Off
```

Comment:
Don't make the rules more complicated by introducing options here. Just leave it as off (the head ref can see the gamestate on the GSV/TCM screen anyways)? Other opinions?

Reply:
I make it optional for now because we are late in the season, but at the same time I think it's great to the option to differentiate every state on the robot.  Suggestion for next years should be to make it mandatory.

Reply:
I agree with Stefan - let's go with one option. I vote for Cyan. I think it will be helpful to teams to distinguish the states.